web/models/workflow.ex

```diff
@@ -40,4 +40,14 @@ defmodule Designator.Workflow do
     |> cast(params, [:id, :configuration, :grouped, :prioritized])
   end
 
+  def sort_decimal_values(result_tuples) do
+    # sort_by/3 takes the following params
+    # result set param for sorting
+    # a mapper function to unpack the tuple index [1] decimal values
+    # and a comparison function to sort on ensuring ascending order (elem 1 < elem 2)
+    # https://hexdocs.pm/decimal/1.9.0/readme.html#comparisons
+    Enum.sort_by(result_tuples, &elem(&1, 1), &Decimal.cmp(&1, &2) != :gt)
```

Comment:
Ah this was something I was looking into. It looked like Decimal comparison was fixed in the most recent version 2.0.0, but I ran into dependency conflict issues. nice.